sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/LineLength.vb

```diff
@@ -6,8 +6,12 @@ Namespace Tests.Diagnostics
     Public Class LineLength
         Public Sub LineLength()
 
-            Console.WriteLine("This is OK...") ' With these comments...........................................................
-            Console.WriteLine() ' Noncompliant   {{Split this 128 characters long line (which is greater than 127 authorized).}}
+            Console.WriteLine("This is OK...") ' With these comments............................................................
+            ' Noncompliant@-1 {{Split this 128 characters long line (which is greater than 127 authorized).}}
+            Console.WriteLine()
+
+            ' Next line is compliant because it contains the error pattern so we prevent the issue to be reported
+            Console.WriteLine(" error: foo bar oeirpo worweoi rewopir epwooriewporiewpor iweporiw oriewporiewprowe ipoewirpewoirewporiwepo")

```

Comment:
The comment is very useful.

The test doesn't quite belong here - perhaps worth extracting to a separate test and run couple of rules? Otherwise, it could be very confusing when we look at that in few months.